### ISSUE TITLE
is image build in linux?

### ISSUE THREAD

**Issue by GoodJoey** [EdwardZhang88/MXonK8S#1]
thanks #

**Comment by EdwardZhang88** [comment]
It could be that I mistakenly uploaded the MacOS version. You can try docker pull helloworld88/mx_operator. If that still doesn't work, I suggest you compile directly from this repo  https://github.com/deepinsight/mxnet-operator.

**Comment by GoodJoey** [comment]
@EdwardZhang88  
hi, may i ask what's your k8s version?
run on k8s 1.9, scheduler get some errors.

or maybe it's my mxnet image issue? where can i download mxnet-gpu-cuda9-dist:0.12.0?

**Comment by GoodJoey** [comment]
@EdwardZhang88 
would you please tell me the version your k8s is ?
my k8s has some issue when doing the schedule binding, it will failed in this step: "src/van.cc:75: Bind to role=scheduler, id=1, ip=mxnet-cluster-mnist-scheduler-qhst-0".

**Comment by EdwardZhang88** [comment]
Hi, I used to run mxjob on K8S1.8. We have recently upgraded to 1.9 and mxjob is still running OK. There could be multiple reasons why you got the error. In my case, I have had issues like scheduler port was occupied and server(PS) got disconnected for unknown reason and scheduler seemed to be unable to re-check in the new server, etc. One useful tip is to set os.environ.update({"PS_VERBOSE": "2"}) and you will be able to see more detail logs regarding the communication within cluster. I used to follow this blog and found [it](https://blog.kovalevskyi.com/mxnet-distributed-training-explained-in-depth-part-1-b90c84bda725) really useful. Hope that helps.
